.envrc

```diff
@@ -88,7 +88,9 @@ export PATH
 export FISSILE_CONFIG_PREFIX=scf
 
 # We will persist compilation results to the VM here
-export SCF_PACKAGE_COMPILATION_CACHE="${SCF_PACKAGE_COMPILATION_CACHE:-${PWD}/.fissile/compilation}"
+if [ -z ${SCF_PACKAGE_COMPILATION_CACHE+x} ]; then
+  export SCF_PACKAGE_COMPILATION_CACHE="${PWD}/.fissile/compilation"
+fi
 
```

Comment:
Isn't the idiomatic way to write this:

```
export SCF_PACKAGE_COMPILATION_CACHE="${SCF_PACKAGE_COMPILATION_CACHE:-${PWD}/.fissile/compilation}"
```

Finally, the functionality doesn't seem to match the PR description: you don't disable the cache, you just point it to a different location.

Reply:
The fact that I was confused about this shows that at least additional documentation would be helpful. Using a separate variable to disable caching would be even clearer.

Reply:
I agree, bash is making not making this intuitive to read.
There's a `SCF_DISABLE_PACKAGE_COMPILATION_CACHE` now.